source/prerequisite-tasks.js

```diff
@@ -21,8 +21,8 @@ module.exports = (input, pkg, options) => {
 			task: async () => {
 				const versions = JSON.parse(await execa.stdout('npm', ['version', '--json']));
 
-				if (version(versions.npm).satisfies('<6.8.0')) {
-					throw new Error('Please upgrade to npm@6.8.0 or newer');
+				if (!version(versions.npm).satisfies('<=6.5.0 || >=6.8.0')) {
```

Comment:
```suggestion
				if (version(versions.npm).satisfies('>6.5.0 && <6.8.0')) {
```

Totally optional, but it makes things more readable (in my opinion).

Reply:
@itaisteinherz I would prefer that too but seems like semver only supports logical or

Reply:
I opened an issue in [`semver`](https://github.com/npm/node-semver) to add support for logical ANDs: npm/node-semver#273.

Reply:
_Note for the future: you can add a logical AND when using `semver` by simply adding a space between two ranges (in this case, `>6.5.0 <6.8.0`)._